### PR TITLE
Deploy metrics for the linuxptp-daemon

### PR DESCRIPTION
**Pull request by SchSeba**
Added metrics for the linuxptp-daemon

Signed-off-by: Sebastian Sch <sebassch@gmail.com>

### CODE REVIEW

File: bindata/linuxptp/ptp-daemon.yaml [hunk 1]
```diff
@@ -40,7 +40,103 @@ spec:
...
           configMap:
             name: ptp-configmap
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    name: monitor-ptp
+  annotations:
+    networkoperator.openshift.io/ignore-errors: ""
```
Comment (zshi-redhat) [inline bindata/linuxptp/ptp-daemon.yaml]: This looks like legacy from cluster-network-operator.

File: deploy/namespace.yaml [hunk 2]
```diff
@@ -6,3 +6,4 @@ metadata:
   labels:
     name: openshift-ptp
     openshift.io/run-level: "1"
+    openshift.io/cluster-monitoring: "true"
```
Comment (zshi-redhat) [inline deploy/namespace.yaml]: Please also update the same for 4.4 manifest which is used when deploying ptp-operator via operator hub.
https://github.com/openshift/ptp-operator/blob/master/manifests/4.4/ptp-operator.v4.4.0.clusterserviceversion.yaml#L112

File: deploy/clusterrole.yaml [hunk 3]
```diff
@@ -33,6 +33,18 @@ rules:
 - apiGroups: ["ptp.openshift.io"]
   resources: ["*"]
   verbs: ["*"]
+- apiGroups: ["monitoring.coreos.com"]
```
Comment (zshi-redhat) [inline deploy/clusterrole.yaml]: Please also update the same for 4.4 manifest - install.spec.clusterPermissions`: https://github.com/openshift/ptp-operator/blob/master/manifests/4.4/ptp-operator.v4.4.0.clusterserviceversion.yaml#L129

**Comment by SchSeba** [comment]
@zshi-redhat thanks for the comments!

Done!

File: manifests/4.4/ptp-operator.v4.4.0.clusterserviceversion.yaml [hunk 4]
```diff
@@ -118,6 +118,8 @@ spec:
       metadata:
         name: ptp-operators
         namespace: openshift-ptp
+        labels:
+          openshift.io/cluster-monitoring: "true"
```
Comment (zshi-redhat) [inline manifests/4.4/ptp-operator.v4.4.0.clusterserviceversion.yaml]: This shall be added in above Namespace section.

Comment (SchSeba) [reply manifests/4.4/ptp-operator.v4.4.0.clusterserviceversion.yaml]: Sorry my bad! 

Thanks, done!

**Pull request merged**